Framework/Core/src/WorkflowSerializationHelpers.cxx

```diff
@@ -640,7 +647,7 @@ void WorkflowSerializationHelpers::dump(std::ostream& out,
 
     w.Key("inputs");
     w.StartArray();
-    for (auto& input : processor.inputs) {
+    for (auto const& input : processor.inputs) {
```

Comment:
Do we really need all those `const`? `auto&` should already be such because workflow itself is `const`.

Reply:
Not necessary, of course, but easier to read IMO. Should I remove it?

Reply:
Yes, please, @chiarazampolli and myself are changing something in the same file and I prefer to minimise change.

Reply:
Done.